### PR TITLE
Add version info to windows main exe resource

### PR DESCRIPTION
**Pull request by albertony**
Continuing from https://github.com/debauchee/barrier/pull/978, now with main GUI application exe also.

### CODE REVIEW

File: src/cmd/barrierd/barrierd.rc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Microsoft Visual C++ generated resource script.
 //
-#include "resource.h"
+//#include "resource.h"
```
Comment (p12tic) [inline src/cmd/barrierd/barrierd.rc:3]: Any reason for commenting this out versus removing the line?

**Review by p12tic — CHANGES_REQUESTED** [review]
Looks good, just a single nitpick.

File: src/cmd/barrierd/barrierd.rc (hunk 1 above)
Comment (albertony) [reply src/cmd/barrierd/barrierd.rc:3]: No, not really... I can remove it, for sure.

**Comment by albertony** [comment]
> Looks good, just a single nitpick.

I like nitpicking!

**Review by p12tic — APPROVED** [review]

**Pull request merged**